docs/intro-Cloud/gpus.md

```diff
@@ -1,27 +1,34 @@
 # GPUs
 
 VSC Tier-1 Cloud users can also deploy VMs with different kind of GPUs.
-A full GPU card is connected directly to the VM via [PCI passthrough](https://libvirt.org/docs/libvirt-appdev-guide-python/en-US/html/libvirt_application_development_guide_using_python-Guest_Domains-Device_Config-PCI_Pass.html) and it is not shared between VMs.
+A full GPU card is connected directly to the VM via
+[PCI passthrough](https://libvirt.org/docs/libvirt-appdev-guide-python/en-US/html/libvirt_application_development_guide_using_python-Guest_Domains-Device_Config-PCI_Pass.html)
+and it is not shared between VMs.
 
-See section [Instance types and flavors](./ch_Flavors.md#instance-types) for more information about the different GPUs available (`GPUv*` flavors).
+See section [Instance types and flavors](../flavors/index.html#instance-types-and-flavors)
```

Comment:
doesn't `flavors#instance-types-and-flavors` work?

Reply:
@boegel fixed, removed hardcoded links